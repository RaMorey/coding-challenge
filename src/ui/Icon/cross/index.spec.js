import { shallow } from 'enzyme';
import React from 'react';
import CrossIcon from '.';

describe('CrossIcon component', () => {
  it('renders correctly with default props', () => {
    const component = shallow(<CrossIcon />);
    expect(component).toMatchSnapshot();
  });
});
