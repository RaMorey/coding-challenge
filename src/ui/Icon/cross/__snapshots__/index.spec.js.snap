// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CrossIcon component renders correctly with default props 1`] = `
<svg
  aria-hidden={true}
  height={16}
  viewBox="6 6 12 12"
  width={16}
>
  <path
    d="M7.324 6L6 7.324l4.715 4.716-4.677 4.676 1.205 1.204 4.677-4.677L16.675 18 18 16.675l-4.756-4.755 4.675-4.676-1.203-1.204-4.677 4.675L7.324 6z"
  />
</svg>
`;
